jetstream/cli.py

```diff
@@ -296,27 +347,36 @@ def export_statistics_to_json(project_id, dataset_id, bucket):
     export_statistics_tables(project_id, dataset_id, bucket)
 
 
-@cli.command("rerun_config_changed")
+@cli.command()
 @project_id_option
 @dataset_id_option
-def rerun_config_changed(project_id, dataset_id):
+@argo_option
+@zone_option
+@cluster_id_option
+@monitor_status_option
+def rerun_config_changed(project_id, dataset_id, argo, zone, cluster_id, monitor_status):
```

Comment:
do we need a non-Argo version of this?

Reply:
Not sure if we need it, we might want to keep it if we for some reason want to test something locally?